hugegraph-store/hg-store-rocksdb/src/main/java/org/apache/hugegraph/rocksdb/access/RocksDBSession.java

```diff
@@ -430,6 +430,9 @@ private void openRocksDB(String dbDataPath, long version) {
             List<byte[]> columnFamilyBytes = RocksDB.listColumnFamilies(new Options(), dbPath);
 
             ColumnFamilyOptions cfOptions = new ColumnFamilyOptions();
+            if(this.hugeConfig.get(RocksDBOptions.DISABLE_AUTO_COMPACTION)){
```

Comment:
consider importing the `.editorconfig` configuration file located at the root of the project for code formatting